packages/gestalt/src/Row.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import FlexBox from './FlexBox.js';
+import Box, {
+  AlignContentPropType,
+  AlignItemsPropType,
+  AlignSelfPropType,
+  DimensionPropType,
+  FlexPropType,
+  JustifyContentPropType,
+  MarginPropType,
+  OverflowPropType,
+  PaddingPropType,
+  type AlignContent,
+  type AlignItems,
+  type AlignSelf,
+  type Dimension,
+  type Flex,
+  type JustifyContent,
+  type Margin,
+  type Overflow,
+  type Padding,
+} from './Box.js';
+
+type Props = {|
+  alignContent?: AlignContent,
+  alignItems?: AlignItems,
+  alignSelf?: AlignSelf,
+  children?: React.Node,
+  fit?: boolean,
+  flex?: Flex,
+  height?: Dimension,
+  justifyContent?: JustifyContent,
+
+  maxHeight?: Dimension,
```

Comment:
@dangerismycat should we limit the API of `Row` / `Stack` even more? Just keeping it to the absolute necessary flex props. If people need to set width / height / padding / overflow, they can always wrap it in a `<Box />`
Mainly trying to avoid the issue we have with `Box` where the API is too wide.

```
  alignContent?: AlignContent,
  alignItems?: AlignItems,
  alignSelf?: AlignSelf,
  children?: React.Node,
  direction?: Direction,
  fit?: boolean,
  flex?: Flex,
  justifyContent?: JustifyContent,
  gap?: Margin,
  wrap? boolean,
```

Reply:
Sure, limiting the API has been a goal of mine. I wasn't sure where to draw the line. You think the above list of props should be all that is available?

Reply:
@dangerismycat Yes, I think that would be a good start - we can always add more of them later (taking away is harder).

Reply:
Also removing `direction`, as that's one of the props we're hard-coding as part of Row/Stack